principles.md

```diff
@@ -0,0 +1,47 @@
+# datavaluepeople principles and their trade-offs
+
+- Never hire under a hard time pressure:
+    - Why: Because who is in dvp is _the_ most important and defining thing so decisions on who to
+      work with should never affected by external influence.
+    - Downside: We have to turn down work if we don't have capacity.
+- Prolific in our engagements but careful in our commitments:
+    - Why: The combination of the two will help us quickly, and in a low-risk way, find work and
+      people we're stoked about.
+    - Downside: We are slower to grow. We spend time and energy on "stuff" that isn't necessarily
+      turn into a commitment
+- Partners are equal:
+    - Why: We believe ownership is integral to motivation, so we want all partners to have equal
+      ownership.
+    - Downside: Slower to make decisions. Possible deadlocks. Requires more energy in communication
+      and more maintaining of relationships.
+- Members have 100% opportunity but require 0% commitment:
+    - Why: The people we want to be in dvp know their own worth. Such people don't like to be/are
+      hard to constrained. We want to demonstrate we a collaborative group not an extractive one.
+    - Downside: Members are a less reliable capacity for projects.
+- Structure engagements with/within dvp to incentivise further investment and engagement.
+    - Why: We don't want to create lock-ins, but we do want to foster deeper cooperation with
+      partners/members/clients/partnerships. A main structural decision to achieve this are Project
+      Pots.
+    - Downside: Slower to form more reliable and strong collaborators.
+- We are fully remote by default:
+    - Why: Goes without saying.
+    - Downside: See the professed downsides by old large employers.
+- We strive to work as asynchronously as possible:
+    - Why: We believe this is more efficient, enables more diverse work styles, and makes for a
+      more flexible work life and enjoyable life.
+    - Downside: Requires more energy in communicating precisely. Potential members may not be used
+      to working this way.
+- We choose pragmatic win-wins over squeezing engagements for all we can get:
+    - Why: Collaboration and mutual wins feel better, and we think give better returns for us in
+      the long run.
+    - Downside: We might earn lower rates than we otherwise could.
+- We leave a wake/trail which others can take up:
```

Comment:
Ideas from Dan Ben Andrew:
Maybe we want to add another/replace this with one that is more explicit on working, especially in dev work, in a way in which it make life easy and enjoyable for others. E.g. PRs, commit messages, communicating clearly. Prioritizing the reader/reviewer of code at the expense of de-prioritizing the writer of the code. 
We should have in mind the enjoyment of the reviewer when creating PRs.
PRs should be easy as pos for the review.